src/constants.py

```diff
@@ -5,8 +5,7 @@
 class VerbosityLevel(enum.IntEnum):
     """ Define levels of output verbosity"""
     LOW = 0
-    MEDIUM = 1
-    HIGH = 2
+    HIGH = 1
```

Comment:
No MEDIUM anymore? I guess if it's not used.

Reply:
I didn't have a use for it yet, so I figured it should be omitted until there is a proper use.